packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -77,28 +82,28 @@ export class OrgAuthInfo {
       return JSON.stringify(defaultDevHubUserName).replace(/\"/g, '');
     } catch (err) {
       console.error(err);
-      telemetryService.sendException(
-        'get_default_devhub_username_alias',
-        err.message
-      );
+      if (err instanceof Error) {
+        telemetryService.sendException(
+          'get_default_devhub_username_alias',
+          err.message
+        );
+      }
       return undefined;
     }
   }
 
   public static async getUsername(usernameOrAlias: string): Promise<string> {
-    return (await Aliases.fetch(usernameOrAlias)) || usernameOrAlias;
+    const info = await StateAggregator.getInstance();
+    return info.aliases.getUsername(usernameOrAlias) || usernameOrAlias;
+    // return (await Aliases.fetch(usernameOrAlias)) || usernameOrAlias;
   }
 
   public static async isAScratchOrg(username: string): Promise<boolean> {
-    try {
```

Comment:
Interesting to unwrap this; curious what Gordon's logic might have been here.

Reply:
Looks like a try/catch isn't needed since it only re-throws the error.